packages/tables/src/Columns/Layout/Panel.php

```diff
@@ -8,6 +8,8 @@ class Panel extends Component
 {
     protected string $view = 'tables::columns.layout.panel';
 
+    protected Closure | bool $collapsed = true;
```

Comment:
This needs to be named `$isCollapsed`, and the `bool` always comes before `Closure`.